README.md

```diff
@@ -48,11 +48,13 @@ First, make sure your DDEV version is at least 1.23.0. Next, create a new folder
 for your project and `cd` into it. Then:
 
 ```
-$ ddev config --project-type=drupal --php-version=8.3
-$ ddev start
+$ ddev config --project-type=drupal11 --php-version=8.3
```

Comment:
Removing `ddev start` is OK here, but as an instructional matter I usually consider it better for something like a README to show the actual command, even though `ddev composer create` is happy to start for you.